language/move-lang/tests/move_check/parser/expr_abort_missing_value.exp

```diff
@@ -3,9 +3,9 @@ error:
    ┌── tests/move_check/parser/expr_abort_missing_value.move:5:5 ───
    │
  5 │     }
-   │     ^ Unexpected token: '}'
+   │     ^ Unexpected '}'
    ·
  5 │     }
-   │     - Expected: 'move', 'copy', 'break', 'continue', '[Name]', '[Address]', 'true', 'false', '[U64]', '(', '{', '::'
+   │     - Expected an expression term
```

Comment:
Maybe just 'Expected an expression' ?

Reply:
But its a bit hard since the error occurs during an expression. Maybe unexpected end of an expression? unsure

Reply:
I don't think either of those work. This can happen right at the start of an expression, so "unexpected end" doesn't make sense. It's also not just a plain "expression" because this is a subset of things that compose an expression (contrast it with a UnaryExp). Maybe we document this terminology (or make up a different name) when we document the language?